ant-service-management/src/node/node_service_data_v1.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) 2024 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use super::NodeServiceData;
+use crate::{error::Result, ServiceStatus};
+use ant_bootstrap::InitialPeersConfig;
+use ant_evm::{AttoTokens, EvmNetwork, RewardsAddress};
+use ant_logging::LogFormat;
+use libp2p::{Multiaddr, PeerId};
+use serde::{Deserialize, Deserializer, Serialize};
+use std::{
+    net::{Ipv4Addr, SocketAddr},
+    path::PathBuf,
+};
+
+pub const NODE_SERVICE_DATA_SCHEMA_V1: u32 = 1;
+
+fn schema_v1_value() -> u32 {
+    NODE_SERVICE_DATA_SCHEMA_V1
+}
+
+#[derive(Clone, Debug, Serialize)]
+pub struct NodeServiceDataV1 {
```

Comment:
Just a tiny optional thing. Is it worth maybe just using a small comment to point out which fields are new/different in V1?

Reply:
Updated it!